CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.0.1 - 2020-06-15
+
+Fix:
+
+- Fix deep nested dynamic types validation
```

Comment:
I wonder if we should start tracking the issues on CHANGELOG from here on? What do you think?